src/app/modules/categories/movie/movie.component.scss

```diff
@@ -0,0 +1,29 @@
+#single-movie {
```

Comment:
don't use ids for styling, use classes instead

Reply:
Why?